packages/core/src/client.ts

```diff
@@ -142,6 +142,11 @@ export interface TokenResponse {
   readonly profile: Reference<ProfileResource>;
 }
 
+export interface BotEvent {
+  readonly contentType: string;
+  readonly input: any;
```

Comment:
Can we be more explicit here? e.g. `Resource | string` to start off with?

Reply:
or i guess `Resource | HL7Message | string` (not sure what we're doing for ASTM )